src/RequestExecutor/HttpRequestExecutor.ts

```diff
@@ -308,8 +309,8 @@ export class HttpRequestExecutor implements RequestExecutor {
   private async parseBody(
     res: IncomingMessage,
     options: {
-      maxBodySize: number;
-      requiresTruncating: boolean;
+      maxSize: number;
+      allowTruncation: boolean;
       decompress: boolean;
     }
   ): Promise<Buffer> {
```

Comment:
the idea for https://github.com/NeuraLegion/bright-cli/pull/585#issuecomment-2276164092
then you can log warning in `truncateResponse` where mime stuff is handled(thus keeping it scoped to single function)

```suggestion
  ): Promise<{body: Buffer, transform: 'truncated' | 'omitted' | false}> {
```